src/core/config.go

```diff
@@ -926,6 +953,60 @@ func (config *Configuration) PrintAliases(w io.Writer) {
 	}
 }
 
+// Command retrieves an alias config, constructs a Command from it, and adds it to the parser
+func (a Alias) Command(name string, cmd *flags.Command) {
+	ac := &AliasConfig{}
+	readAliasConfigFile(ac, a.Config)
+
+	// Add alias command and flags
+	var data interface{} = &struct{}{}
+	cmd.AddCommand(name, name, a.Desc, data)
+	addFlags(ac.NestedFlag, cmd)
+
+	// Recurse through nested subcommands
```

Comment:
I think we should set this:
```
	if ac.Command.Cmd == "" {
		cmd.SubcommandsOptional = false
	}
```

That way, we don't do silly things when trying to do `plz sef` without any subcommand.